src/core/resumes/resume.service.ts

```
@@ -3,46 +3,123 @@ import { Injectable } from '@nestjs/common';
+        return new GetResumeResponse(resume);
+    }
+
     async getBestResumes(
         query: PaginationQueryDto,
     ): Promise<GetResumeResponse[]> {
-        const resumes: GetResumeResponse[] =
-            await this.resumeRepository.getBestResumes(query);
+
+        const { offset = 0, limit = 10 } = query;
+
+        const twoWeeksAgo: Date = new Date;
```

ESLint

[error] Insert `()`

(prettier/prettier)